sys/fs/fuse/fuse_vnops.c

```diff
@@ -1372,12 +1372,12 @@ fuse_vnop_link(struct vop_link_args *ap)
 	feo = fdi.answ;
 
 	if (fli.oldnodeid != feo->nodeid) {
+		static const char exterr[] = "Assigned wrong inode for a hard link.";
```

Comment:
This error means that the FUSE server did something wrong, not the file system user.  The `fuse_warn` macro makes that clear.  But ktrace won't.  So let's adjust the message to make it clear for both:
```suggestion
		static const char exterr[] = "Server assigned wrong inode for a hard link.";
```